packages/neo-one-server-client/src/ServerManager.ts

```diff
@@ -171,6 +196,15 @@ export class ServerManager {
     await fs.writeFile(this.pidPath, `${pid}`);
   }
 
+  public async writeVer(ver: string): Promise<void> {
+    await fs.ensureDir(path.dirname(this.verPath));
+    await fs.writeFile(this.verPath, `${ver}`);
+  }
+
+  public get verPath(): string {
```

Comment:
Let's not abbreviate version, change all usages of `ver` in this file to `version`. (The `.ver` file name can stay though)

Reply:
👍